TriangleTest/TasksTest.cs

```diff
@@ -62,8 +62,28 @@ private static List<ColoredTriangle> ProvideTriangles()
                 ColoredTriangle.Parse("Orange-6:Yellow-9:Green-4"),
                 ColoredTriangle.Parse("Orange-10:Orange-25:Orange-20"),
                 ColoredTriangle.Parse("Red-6:Green-8:Blue-3"),
+                ColoredTriangle.Parse("Red-4:Blue-4:Blue-4"),
                 ColoredTriangle.Parse("Green-14:Green-20:Green-7")
             };
         }
+
+        [Test]
+        public void ColoringSideTest()
+        {
+            var triangles = ProvideTriangles();
+            var coloredTriangles = Tasks.ColoringSide(triangles);
+            
+            Assert.AreEqual(coloredTriangles.Count, 3);
+        }
+
+        [Test]
+        public void TestWriteTriangleListToFile()
+        {
+            //var triangles = ProvideTriangles();
+            const string DataDir = "../../Data";
+            var triangle = Tasks.ReadTrianglesToSortedList($"{DataDir}/Triangles.txt");
```

Comment:
In this test we are only testing the write function.
Replace this with `ProvideTriangles()` function